### PR TITLE
Add guide how to create a custom asset - Closes #887

### LINKED ISSUE #887: Add guide how to create a custom asset
Parent issue: #841

### PR DESCRIPTION
**Pull request by Tschakki**
Closes #887

### CODE REVIEW

File: modules/ROOT/pages/guides/app-development/asset.adoc [hunk 1]
```diff
@@ -0,0 +1,230 @@
...
+const { BaseAsset } = require('lisk-sdk');
+
+class MyAsset extends BaseAsset {
+
+}
+
+module.exports = MyAsset;
+----
+
+== Setting name and ID of the module
```
Comment (ManuGowda) [inline modules/ROOT/pages/guides/app-development/asset.adoc]:
```suggestion
== Setting name and ID of the asset
```

File: modules/ROOT/pages/guides/app-development/asset.adoc [hunk 2]
```diff
@@ -0,0 +1,230 @@
...
+
+Inside of the `MyAsset` class, define the different properties of the transaction asset:
+
+`id`(number)::
+The ID of the asset.
```
Comment (ManuGowda) [inline modules/ROOT/pages/guides/app-development/asset.adoc]:
```suggestion
The ID of an asset.
```

File: modules/ROOT/pages/guides/app-development/asset.adoc [hunk 3]
```diff
@@ -0,0 +1,230 @@
...
+Must be unique within the module.
+The ID is used for example to for creation of transactions of the module.
```
Comment (ManuGowda) [inline modules/ROOT/pages/guides/app-development/asset.adoc]: Not able to understand the sentence here, could be framed better?

File: modules/ROOT/pages/guides/app-development/asset.adoc [hunk 4]
```diff
@@ -0,0 +1,230 @@
...
+`name`(string)::
+The name of the asset.
```
Comment (ManuGowda) [inline modules/ROOT/pages/guides/app-development/asset.adoc]:
```suggestion
The name of an asset.
```

File: modules/ROOT/pages/guides/app-development/asset.adoc [hunk 5]
```diff
@@ -0,0 +1,230 @@
...
+
+class MyAsset extends BaseAsset {
+  id = 0; <1>
+  name = 'myAsset';
+}
+
+module.exports = MyAsset;
+----
+
+<1> The transaction asset `MyAsset` is the first asset of the module `MyModule`, therefore we set the `id` to the lowest possible value, which is `0`.
```
Comment (ManuGowda) [inline modules/ROOT/pages/guides/app-development/asset.adoc:80]: just so you know, the id must be uniq within the module, but the value can be lowest or highest unless it uniq its valid.

**Review by ManuGowda — DISMISSED** [review]

File: modules/ROOT/pages/guides/app-development/asset.adoc (hunk 3 above)
Comment (Tschakki) [reply modules/ROOT/pages/guides/app-development/asset.adoc]:
```suggestion
The ID is used (combined with the module ID)  for example to create transaction objects.
```

**Pull request merged**